src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM/help-serverId.html

```diff
@@ -1,4 +1,4 @@
 <div>
-    Select a Bitbucket Server from the list. If you cannot see your server listed, check the System Configuration
-    (Manage Jenkins -> Configure System).
+    This is the Bitbucket instance that contains the project you want Jenkins to build from. If you cannot see your
```

Comment:
/nit "contains the _repository_" 
(because projects are just a container for the things that Jenkins actually needs access to)